README.md

```diff
@@ -31,12 +31,13 @@ List of dependencies:
  - yojson >= 1.6.0
  - cmdliner
  - ppx\_inline\_test
+ - why3
 
 Using Opam, a suitable OCaml environment can be setup as follows.
 ```bash
 opam switch 4.05.0
 eval `opam config env`
-opam install dune odoc menhir yojson cmdliner bindlib.5.0.0 timed.1.0 earley.2.0.0 ppx_inline_test
+opam install dune odoc menhir yojson cmdliner bindlib.5.0.0 timed.1.0 earley.2.0.0 ppx_inline_test why3
```

Comment:
Same here, add version requirement (ass used in Travis script).

Reply:
You mean we specify the version directly? as done in b8fb9af3a00c2fbb135c1dbd2e38fb315051079c ?

Reply:
Yes, exactly.